### PR TITLE
Make Logger Public

### PR DESCRIPTION
**Pull request by a76yyyy**

### CODE REVIEW

File: logging/logrus/logger.go [hunk 1]
```diff
@@ -45,96 +45,96 @@ func NewLogger(opts ...Option) *Logger {
...
 
 	return &Logger{
-		l: cfg.logger,
+		L: cfg.logger,
 	}
 }
 
 func (l *Logger) Logger() *logrus.Logger {
-	return l.l
+	return l.L
```
Comment (CoderPoet) [inline logging/logrus/logger.go:53]: Why would you want to change it to public? If you need to get original logger instance, is it enough to use the logger() method?

Comment (a76yyyy) [reply logging/logrus/logger.go:53]: When I use zapLogger.NewLogger, I can't get SugaredLogger in Logger struct

Comment (a76yyyy) [reply logging/logrus/logger.go:53]: I don't use logrus, so the modification of logrus is not strict, maybe you can provide a more effective way

Comment (rogerogers) [reply logging/logrus/logger.go:53]: Maybe we can expose this via a method instead of making it public. By the way, why use sugar directly?

Comment (a76yyyy) [reply logging/logrus/logger.go:53]: Well, I think the #16 PR  seems to be able to solve my problem

Comment (rogerogers) [reply logging/logrus/logger.go:53]: > Well, I think the #16 PR seems to be able to solve my problem

Ok, you need the kv log methods? We will focus on issue 16.

**Pull request closed without merging**